docs/additional-functionality/cache-serializer.md

```diff
@@ -29,22 +29,30 @@ nav_order: 2
   `spark.sql.inMemoryColumnarStorage.enableVectorizedReader` will not be honored as the GPU
   data is always read in as columnar. If `spark.rapids.sql.enabled` is set to false
   the cached objects will still be compressed on the CPU as a part of the caching process.
-  Also note that Parquet doesn't support CalendarIntervalType or NullType out of the box, but
-  ParquetCachedBatchSerializer does by decomposing intervals to struct containing the
-  months, days and microseconds and NullType to Int column containing nulls.
+  
+  Please note that ParquetCachedBatchSerializer doesn't support negative decimal scale, so if 
+  `spark.sql.legacy.allowNegativeScaleOfDecimal` is set to true ParquetCachedBatchSerializer
+  should not be used.  Using the serializer with negative decimal scales will generate
+  an error at runtime.
 
-  Please make sure to use the right package corresponding to the spark version you are using. To use
+  Make sure to use the right package corresponding to the spark version you are using. To use
   this serializer with Spark 3.1.1 please run Spark with the following conf.
   ```
   spark-shell --conf spark.sql.cache.serializer=com.nvidia.spark.rapids.shims.spark311.ParquetCachedBatchSerializer"
   ```
-  Please see the below table for all the names of the serializers corresponding to the Spark
+  See the below table for all the names of the serializers corresponding to the Spark
   versions
-
+ 
   | Spark version | Serializer name |
   | ------ | -----|
   | 3.1.1 | com.nvidia.spark.rapids.shims.spark311.ParquetCachedBatchSerializer |
   | 3.1.2 | com.nvidia.spark.rapids.shims.spark312.ParquetCachedBatchSerializer |
-  | 3.2.0 | com.nvidia.spark.rapids.shims.spark320.ParquetCachedBatchSerializer |
+  | 3.1.3 | com.nvidia.spark.rapids.shims.spark313.ParquetCachedBatchSerializer |
+  | 3.2.0 | com.nvidia.spark.rapids.shims.spark320.ParquetCachedBatchSerializer | 
+  
+##          Supported Types                       
+ 
+ All types are supported on the CPU, on the GPU, ArrayType, MapType and BinaryType are not
```

Comment:
The fix from Gpu -> GPU is not on branch-21.06.  Normally these kind of PRs don't fixup docs, they should just merge the changes in the docs to the gh-pages branch.  Patching things "on the fly" in these PRs can easily leave those same fixes missing from the development branches, and anyone looking at the documentation on those branches will not see those fixes.  There should be a followup to fix this in branch-21.08.

Reply:
Added PR #2840 for the documentation fix.